test/functional/fixtures/blank-page.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Title</title>
+</head>
+<body>
+
+</body>
+</html>
```

Comment:
I think we should separated tests and markup files. We can create a `test/functional/views` dir.